src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -648,7 +765,26 @@ public int ExportSlhDsaPublicKey(Span<byte> destination)
         }
 
         /// <summary>
-        ///  Exports the current key in the FIPS 205 secret key format.
+        ///   Exports the public-key portion of the current key in the FIPS 205 public key format.
+        /// </summary>
+        /// <returns>
+        ///   The public key.
```

Comment:
This overly general wording is why I included SlhDsa in the name of the method.  SPKI is also "the public key", or another SlhDsa instance that has done an import of a public key format.

It's probably best to just stick with what the summary says and change this to

```suggestion
        ///   The FIPS 205 public key.
```

We can also call it in one or both places the "SLH-DSA public key" (nomenclature from FIPS 205 9.1 and Figure 13) or put all the cards on the table as "FIPS 205 SLH-DSA public key".  But just "public key" is definitely lacking.